solc/CommandLineInterface.cpp

```diff
@@ -716,16 +716,16 @@ bool CommandLineInterface::processInput()
 			}
 			if (!isAllowed)
 				return ReadCallback::Result{false, "File outside of allowed directories."};
-			else if (!boost::filesystem::exists(path))
```

Comment:
Is not using the `else if` statements here is just a matter of style?

Reply:
Yes, it is. These changes were part of the original PR and I left them in. I might as well remove them, though...

Reply:
IMHO it does look nicer with the changes, though.

Reply:
I don't mind, was just curious.